### PR TITLE
Add CSV upload capability to Playground, refactor API more

### PR DESCRIPTION
**Pull request by macvogelsang**
@bherr2 I only was able to add CSV upload functionality to the playground. I started trying to integrate the component into the compare form, but it was turning out to be too much work for me to finish in a week. I'll create a new issue and link to my WIP branch so I or someone else can pick it up in the future.

### CODE REVIEW

**Comment by bherr2** [comment]
Ok how solid is the playground version?

**Comment by bherr2** [comment]
If it works well we can merge that feature and setup an issue to have the same support in the compare feature later.

**Comment by macvogelsang** [comment]
We were also talking about doing a security review of the upload functionality in standup yesterday. I have basic file type and file size validation, but technically any client can still spoof the file type and potentially upload malicious code. However, the papa parse serves as an additional check because any file that doesn't meet the strict parsing rules there will just cause the service to throw a 500. Not sure if want some upload security that is more sophisticated than that.

**Comment by bherr2** [comment]
They can do the same with the csvUrl feature.

**Comment by bherr2** [comment]
Number one thing is to limit what touches the raw data and make sure what does is kept up to date (ie papaparse).

File: asctb-api/src/index.ts
Comment (macvogelsang) [inline asctb-api/src/index.ts:171]: @bherr2 By the way, I deleted this old v1 route and never moved it to a new file. I didn't think it was being used anywhere but correct me if I'm wrong.

Comment (bherr2) [reply asctb-api/src/index.ts:171]: Hmm, ok, that's fine. Thanks for letting me know before I merged it.

**Review by bherr2 — APPROVED** [review]

**Pull request merged**
